.github/workflows/pre-main.yaml

```diff
@@ -214,24 +214,26 @@ jobs:
         with:
           ref: ${{ github.sha }}
 
-      # Update the CNF containers, helm charts and operators DB
-
+      # Update the CNF containers, helm charts and operators DB.
       - name: Update the CNF DB
         run: |
-          docker pull ${REGISTRY}/${OCT_IMAGE_NAME}:${OCT_IMAGE_TAG}
           mkdir -p "${GITHUB_WORKSPACE}"/offline-db
-          docker run -v "${GITHUB_WORKSPACE}"/offline-db:/tmp/dump:Z --env OCT_DUMP_ONLY=true ${REGISTRY}/${OCT_IMAGE_NAME}:${OCT_IMAGE_TAG}
+          docker run \
+            --env OCT_DUMP_ONLY=true \
+            --rm \
```

Comment:
Probably unnecessary if after this docker command you issue `docker system prune -a`.

Reply:
Honestly we're probably good with #1320 but these changes can't hurt.

Reply:
@jmontesi `--rm` came first. `podman` has `--rmi` parameter for the image. `docker` doesn't, that is why `docker system prune -a` for image.

Reply:
I meant that `docker system prune -a` removes the container as well, that's why I think `--rm` might be unnecessary there. But it's not a big deal anyway.

Reply:
I got you. Agree on 'But it's not a big deal anyway.' :-) Thanks.